pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -36,6 +38,21 @@ type DB struct {
 	check    *time.Ticker
 }
 
+type StorageMibInfo struct {
+	BwUsageInfo    BwUsageTable
```

Comment:
`Bandwidth`

Reply:
Dev in progress side effect. Cleaned up.